modules/button/css/lib/button-deprecated.scss

```diff
@@ -0,0 +1,386 @@
+$_wdc-btn-deprecated-caret-down-icon: svg-load(
+  '#{$_wdc-btn-icons-path}/wd-icon-caret-down.svg',
+  fill=$wdc-btn-icon-color
+);
+$_wdc-btn-deprecated-caret-down-icon-hover: svg-load(
+  '#{$_wdc-btn-icons-path}/wd-icon-caret-down.svg',
+  fill=$wdc-btn-icon-hover-color
+);
+
+@mixin wdc-btn-icon-inline($_icon) {
```

Comment:
This is still used in the new buttons for dropdown buttons. If this file gets deleted when we complete the deprecation we'll have issues